data/xapleaux/xapleaux outfits.txt

```diff
@@ -0,0 +1,188 @@
+# Copyright (c) 2021 by MasterOfGrey
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+outfit "String Drive"
+	category Systems
+	licenses
+		Xapleaux
+	cost 3500000
+	mass 15
+	thumbnail "outfit/string drive"
+	"energy consumption" 2.5
+	hyperdrive 1
+	"jump fuel" 50
+	"outfit space" -15
+	"scram drive" 1.5
+	description "A string drive continuously hums with the frequency of space-time in its vicinity. By constantly remaining attuned to the threads that link one system to the next it has no need to slow down enough to 'search' for a path to follow. Simply by aligning in the correct direction it can claw its way along any hyperthread in its vicinity even if your ship is traveling at a very significant speed. Consequently it needs very little fuel to initiate the process and is smaller as a result. However, this comes at a cost of continuously draining quite a bit of energy to power the attunement apparatus."
```

Comment:
I sense another outfit name capitalization debate...

Reply:
Lol, it can just follow the convention of whatever is used for the others?
It's basically a souped up scram drive.

Reply:
Hyperdrive, jump drive and scram drive are usually lowercase iirc, but they are used by multiple species. This one is unique to the Xapleaux, so it should be capitalized - which sounds rather confusing. Maybe just the fact it's a drive could mean it shouldn't be capitalized? Even I'm unsure about what the convention would be.

I'd probably leave it as-is because that keeps it similar to HD/JD.

@Amazinite thoughts?

Reply:
I would say capitalized, because it's unique. If we get different jump drives for different races (like I tried in #8721) they would also be capitalized.